modules/oxide/src/less/theme/components/collection/collection.less

```diff
@@ -116,6 +119,10 @@
     user-select: none;
   }
 
+  .tox-collection__item--active {
+    fill: #fff;
```

Comment:
This should not be hard coded it needs to be a variable and probably have it be inherited from something else.